README.md

```diff
@@ -1,2 +1,38 @@
 # EDU-Books-Library
-Project Overview EDU-Book Library is a platform where students who like reading pick different books and for one to pick a book from the library they must be either an EDU student, EDU LF and a EDU staff member.
+## Project Overview 
+EDU-Book Library is a platform where students who like reading pick different books and for one to pick a book from the library they must be either an EDU student, EDU LF and a EDU staff member.
+
+## Required Features
+1. Users can create an account and log in.
+2. All users can see available books in the library.
+Books belong to particular categories ie Technology, Science,Mathematics etc.
+EDU LF should be able to add a new book to the library.
+5. Only an EDU student, EDU LF and a EDU staff member are eligible to take a book from the Library.
+6. Only the EDU LF should be able to delete a book.
+
+## Setup
+ `git clone` Github Repo Edu-Books-Library
+
+## Github repo links
+ https://github.com/Ayesiza/EDU-Books-Library.git
+
+## Pivotal Story links
+ https://www.pivotaltracker.com/n/projects/2440488
+
+## gh-pages link
+
+
+## Technologies Used
+  HTML
+  CSs 
+## Languages
+ Javascript
+
+## ### Author
+
+- Ayesiza Hawah
+- Solomon Adutia
+
+### Acknowledgments
+ 
+ Our thanks goes to Our Team LF Jane Zaluwango, other LFs and Team Strap members
```

Comment:
Good work about adding a READMe please add a link to  your GH-pages

Reply:
I not yet hosted i will add when I host after merging one branch atleast